lib/disability_compensation/providers/document_upload/lighthouse_supplemental_document_upload_provider.rb

```diff
@@ -8,9 +8,25 @@ class LighthouseSupplementalDocumentUploadProvider
 
   STATSD_PROVIDER_METRIC = 'lighthouse_supplemental_document_upload_provider'
 
+  # Maps VA's internal Document Types to the correct document_type attribute for a Lighthouse526DocumentUpload polling
+  # record. We need this to create a valid polling record
+  POLLING_DOCUMENT_TYPES = {
+    'L023' => Lighthouse526DocumentUpload::BDD_INSTRUCTIONS_DOCUMENT_TYPE
+  }.freeze
```

Comment:
Major code smell here, the jobs know about the VA document type code (L023 etc.) and that's what we use in the API call, `Lighthouse526DocumentUpload ` has its own labels for the document type so have some record of what the record we are polling was, but the upload code is confusing enough without having to write these adapter mappings